Potential missing dependency

When I run my tests on the latest release (`0.13.1`) I'm getting the following error:

```
ImportError while importing test module '/home/berislav/Development/personal/pyotr/src/tests/test_server.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
src/tests/test_server.py:7: in <module>
    from pyotr.server import Application
src/pyotr/server.py:10: in <module>
    from openapi_core import create_spec
../../../.virtualenvs/pyotr/lib/python3.7/site-packages/openapi_core/__init__.py:3: in <module>
    from openapi_core.shortcuts import (
../../../.virtualenvs/pyotr/lib/python3.7/site-packages/openapi_core/shortcuts.py:4: in <module>
    from openapi_core.validation.request.shortcuts import (
../../../.virtualenvs/pyotr/lib/python3.7/site-packages/openapi_core/validation/request/shortcuts.py:10: in <module>
    from openapi_core.validation.request.validators import RequestValidator
../../../.virtualenvs/pyotr/lib/python3.7/site-packages/openapi_core/validation/request/validators.py:21: in <module>
    from openapi_core.validation.request.datatypes import (
../../../.virtualenvs/pyotr/lib/python3.7/site-packages/openapi_core/validation/request/datatypes.py:3: in <module>
    from werkzeug.datastructures import ImmutableMultiDict
E   ModuleNotFoundError: No module named 'werkzeug'
```

`werkzeug` is listed in `setup.cfg` under `options.extras_require`, but this error implies that it's used more generally?

--- a/setup.cfg
+++ b/setup.cfg
@@ -29,6 +29,7 @@ install_requires =
     strict_rfc3339
     isodate
     attrs
+    werkzeug
     backports.functools-lru-cache; python_version<"3.0"
     backports.functools-partialmethod; python_version<"3.0"
 tests_require =
@@ -45,7 +46,7 @@ exclude =
 
 [options.extras_require]
 django = django>=2.2; python_version>="3.0"
-flask = werkzeug
+flask = flask
 
 [tool:pytest]
 addopts = -sv --flake8 --junitxml reports/junit.xml --cov openapi_core --cov-report term-missing --cov-report xml:reports/coverage.xml

